.codex/skills/material3-expressive-android/references/overview.md

```
@@ -0,0 +1,28 @@
+# M3 Expressive overview
+
+## Sources
+
+- https://developer.android.com/reference/kotlin/androidx/compose/material3/package-summary
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)